### PR TITLE
Onboarding feedback

### PR DESCRIPTION
**Pull request by William-436**
Removed underlines in "Seller_Profile_FeatureStepDefinitions" step definitions file per Onboarding Task 2 feedback.
Made miscellaneous changes prior to receiving feedback.

### CODE REVIEW

File: MVPSt_2023_Jan/MVPSt_2023_Jan/Pages/HomePage.cs [hunk 1]
```diff
@@ -1,177 +1,276 @@
...
-            IWebElement eddegreeTextbox = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[4]/div/div[2]/div/div/div[2]/div[2]/input"));
-            eddegreeTextbox.SendKeys(Degree);
+            //IWebElement eddegreeTextbox = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[4]/div/div[2]/div/div/div[2]/div[2]/input"));
+            educationdegreeTextbox.SendKeys(Degree);
 
             // select year of graduation from drop-down list
             // create object for new SelectElement class
-            IWebElement yearofgradDropDownElement = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[4]/div/div[2]/div/div/div[2]/div[3]/select"));
-            SelectElement selectayearofgrad = new SelectElement(yearofgradDropDownElement);
+            //IWebElement yearofgradDropDownElement = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[4]/div/div[2]/div/div/div[2]/div[3]/select"));
```
Comment (amanbansalnz) [inline MVPSt_2023_Jan/MVPSt_2023_Jan/Pages/HomePage.cs]: remember to remove comments :)

File: MVPSt_2023_Jan/MVPSt_2023_Jan/Pages/TYSPortalPage.cs [hunk 2]
```diff
@@ -1,45 +1,80 @@
...
             {
                 Assert.Fail("Failed to launch Trade Your Skills portal page", ex.Message);
             }
             // wait for email textbox to exist
-            Wait.WaitForElementToExist(driver, "Name", "email", 1);
+            Wait.WaitForElementToExist("Name", emailTextboxName, 1);
             // find Email address textbox and enter valid Email address
-            IWebElement emailTextbox = driver.FindElement(By.Name("email"));
+            //IWebElement emailTextbox = driver.FindElement(By.Name("email"));
             emailTextbox.SendKeys("wbugan@aol.com");
```
Comment (amanbansalnz) [inline MVPSt_2023_Jan/MVPSt_2023_Jan/Pages/TYSPortalPage.cs:55]: you could pass this through the feature but ill leave that to you to decide

File: MVPSt_2023_Jan/MVPSt_2023_Jan/Utilities/CommonDriver.cs [hunk 3]
```diff
@@ -1,34 +1,50 @@
...
 namespace MVPSt_2023_Jan.Utilities
 {
     public class CommonDriver
     {
-        public IWebDriver driver;
+        public static IWebDriver driver;
 
+        // Nunit Hooks
         //[SetUp]
         //[OneTimeSetUp]
```
Comment (amanbansalnz) [inline MVPSt_2023_Jan/MVPSt_2023_Jan/Utilities/CommonDriver.cs]: uncomment the hooks https://codebun.com/how-to-implement-hooks-in-specflow-c/ have look at that link for hooks, once you uncomment this you don't need to call its in your stepDef. See comment on stepDef class

File: MVPSt_2023_Jan/MVPSt_2023_Jan/StepDefinitions/SellerProfileFeatureStepDefinitions.cs [hunk 4]
```diff
@@ -1,42 +1,47 @@
...
+        }
 
         [Given(@"I have signed into Trade Your Skill portal successfully using email ""([^""]*)"" and password ""([^""]*)""")]
         public void GivenIHaveSignedIntoTradeYourSkillPortalSuccessfullyUsingEmailAndPassword(string Email, string Pswd)
         {
-            // open Chrome browser because of using OpenQA.Selenium.Chrome; statement at top of code
-            driver = new ChromeDriver();
-
-            tysportalPageObj.SigninActions(driver);
+            SetupDriver();
```
Comment (amanbansalnz) [inline MVPSt_2023_Jan/MVPSt_2023_Jan/StepDefinitions/SellerProfileFeatureStepDefinitions.cs]: you don't need to do this as the Hook will execute this for you!

File: MVPSt_2023_Jan/MVPSt_2023_Jan/Pages/HomePage.cs (hunk 1 above)
Comment (William-436) [reply MVPSt_2023_Jan/MVPSt_2023_Jan/Pages/HomePage.cs]: I removed many, many comments from all files but probably overlooked a few

File: MVPSt_2023_Jan/MVPSt_2023_Jan/Pages/TYSPortalPage.cs (hunk 2 above)
Comment (William-436) [reply MVPSt_2023_Jan/MVPSt_2023_Jan/Pages/TYSPortalPage.cs:55]: I prefer to leave the code as it is in order to keep my logic and structure consistent with my understanding of the concepts.

File: MVPSt_2023_Jan/MVPSt_2023_Jan/Utilities/CommonDriver.cs (hunk 3 above)
Comment (William-436) [reply MVPSt_2023_Jan/MVPSt_2023_Jan/Utilities/CommonDriver.cs]: Added Specflow [BeforeScenario] to CommonDriver to perform SetupDriver commands

File: MVPSt_2023_Jan/MVPSt_2023_Jan/StepDefinitions/SellerProfileFeatureStepDefinitions.cs (hunk 4 above)
Comment (William-436) [reply MVPSt_2023_Jan/MVPSt_2023_Jan/StepDefinitions/SellerProfileFeatureStepDefinitions.cs]: Removed call to SetupDriver in Given logic in step definition file

**Pull request merged**